webdriver/tests/bidi/browsing_context/navigation_started/navigation_started.py

```diff
@@ -440,14 +440,15 @@ async def test_with_beforeunload_prompt(
     target_url = url("/webdriver/tests/support/html/default.html", domain="alt")
 
     on_navigation_started = wait_for_event(NAVIGATION_STARTED_EVENT)
-    result = await bidi_session.browsing_context.navigate(
+
+    # Trigger navigation, but don't wait for it to be finished.
```

Comment:
Don't we have to await this task at the end of the test to not leak an ongoing navigation into the next test?

Reply:
done, even though I don't think navigation can be leaked, as the browsing context is closed after each test. Handling beforeunload allowed adding `navigation` assertion.

Reply:
Oh, I missed that these tests use the `new_tab` fixture. As such feel free to drop the last commit and go ahead with your formerly proposed solution. I don't think that we need the additional assertions.